static/sass/_pattern_footer.scss

```diff
@@ -97,12 +97,14 @@
     background-size: 13px 13px;
     border-top: 1px solid $color-mid-light;
     color: $color-dark;
-    font-size: 0.8125rem;
+    font-size: 1.2rem;
     margin-top: 0;
     max-width: inherit;
     padding: $sp-medium 0;
+    padding-inline: 1rem;
```

Comment:
I would suggest you simply change the original padding styling on line 103 to `padding: $sp-medium`